README.md

```diff
@@ -18,3 +18,25 @@ The following assumes the use of `node@>=10`.
 ## Run Tests
 
 `yarn test`
+
+## Generate Merkle Tree
```

Comment:
Maybe add a small paragraph explaining about the random address that is used to obfuscate the merkle root and how `-s seed` relates to it.